pkg/retention/retention.go

```diff
@@ -71,6 +73,7 @@ func internalRetentionCleaner() {
 		} else {
 			DoRetentionBasedDeletion(config.GetCurrentNodeIngestDir(), config.GetRetentionHours(), 0)
 			doVolumeBasedDeletion(config.GetCurrentNodeIngestDir(), 60000, deletionWarningCounter)
+			doInodeBasedDeletion(config.GetCurrentNodeIngestDir(), 85, deletionWarningCounter) // 85% max inode usage
```

Comment:
nit:
1. make 85 a constant near the top of this file (or maybe we want it in server.yaml, but I think a constant is probably fine for now)
2. if we'll only ever call `doInodeBasedDeletion` with this value of 85, then maybe it shouldn't even be a parameter, and the function should use the constant directly. This is largely personal preference though, so you can keep it as a parameter if you like that better